core/autoscaling_test.go

```diff
@@ -1990,7 +1993,7 @@ func TestFindSpotInstanceRequests(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			a := &autoScalingGroup{
-				Group:  &autoscaling.Group{AutoScalingGroupName: aws.String("testASG")},
+				Group:  &autoscaling.Group{AutoScalingGroupName: aws.String("testASG"), DesiredCapacity: createInt64(1)},
```

Comment:
I think the SDK should have a way to define an integer, just like it does with `aws.String`